src/plugins/plugin-hapi.ts

```diff
@@ -20,11 +20,19 @@ import {parse as urlParse} from 'url';
 
 import {PluginTypes} from '..';
 
-import {hapi_16} from './types';
+import {hapi_16, hapi_17} from './types';
 
-type Hapi16Module = typeof hapi_16;
+// Used when patching Hapi 17.
+const ORIGINAL = Symbol();
 
-const SUPPORTED_VERSIONS = '8 - 16';
+type Hapi16Module = typeof hapi_16;
+type Hapi17RequestExecutePrivate = {
+  (this: hapi_17.Request): Promise<void>;
+      [ORIGINAL]?: Hapi17RequestExecutePrivate;
```

Comment:
Indentation seems off. Is this how clang-format does?

Reply:
clang-format seems to not format `type` declarations correctly. But this can changed to an interface, so I'll do that.